doc/release/1.4.0-notes.rst

```diff
@@ -47,6 +47,9 @@ lambda expressions for them.
 
 `scipy.linalg` improvements
 ---------------------------
+The function ``scipy.linalg.subspace_angles(A, B)`` now gives correct
```

Comment:
Don’t we want to add this to the 1.4 wiki page instead of here?

Reply:
So far I've put things both places, didn't know which was preferred. I'm happy to put it there once this is merged so it's in both, or remove it here

Reply:
IIRC the wiki is preferred since we added it because we got tired of dealing with all the merge conflicts for the version in the repo. If that's the case we should probably remove the version in the repo until 1.4 is actually released. (@rgommers thoughts?)

Reply:
yep, wiki is preferred. don't bother to remove it here though if it's ready to merge. in that case just leave it out of the wiki. the main purpose of the wiki page is to easily collect content and then merge it into this file in one go without causing merge conflicts.